### PR TITLE
migrate code to python 3 - ignoring web ui

### PR DESCRIPTION
**Pull request by dsk7**
All unit tests and integration tests pass. No further testing has been done.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-defusedxml==0.4.1
-wsgiref==0.1.2
-zope.interface==4.1.2
+Pyro4
+Twisted
+Whoosh
+argparse
+ijson
+psutil
+pycryptopp
```
Comment (hrzlgnm) [inline requirements.txt:7]: There is no python3 version of that package, I would suggest to remove that dependency.

File: requirements.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+serpent
+defusedxml
+wsgiref
```
Comment (hrzlgnm) [inline requirements.txt:10]: That dependency is only needed for Python < 3.2. I guess we could remove it, as we are targeting Python 3.10

File: requirements.txt (hunk 1 above)
Comment (dsk7) [reply requirements.txt:7]: I've created #248 to tackle that. There's code that uses this library that also needs to be removed.

File: requirements.txt (hunk 2 above)
Comment (dsk7) [reply requirements.txt]: Aye, removed!

**Review by hrzlgnm — APPROVED** [review]

**Pull request merged**